doc/api/buffer.markdown

```diff
@@ -393,18 +393,31 @@ byte from the original Buffer.
     // !bc
 
 
-### buf.indexOf(value[, byteOffset])
+### buf.indexOf(value[, byteOffset, encoding])
```

Comment:
Actually, since `typeof byteOffset === 'number'` and `typeof encoding === 'string'` the arguments should actually be `indexOf(value[, byteOffset][, encoding])`. Sorry about that.